metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -155,6 +160,8 @@ public String toString() {
     private final ClusterControlManager clusterControl;
     private final TimelineHashMap<String, Uuid> topicsByName;
     private final TimelineHashMap<Uuid, TopicControlInfo> topics;
+    private final TimelineHashMap<Integer, List<TopicControlInfo>> topicsByBroker;
```

Comment:
This is a placeholder. Will be replaced with the other PR that you mentioned @cmccabe